tutorials/using-cloud-vpn-with-alibaba-with-redundancy

```diff
@@ -0,0 +1,950 @@
+8. Select a **Next hop** for the route. **VPN tunnel** allows you to select an existing Cloud VPN tunnel as a next hop. 
+9. Click **Create**
+10. Repeat steps for remaining VPN tunnels.
+
+**Important**: You cannot edit or update a route after you create it. To modify a custom static route, you must delete it and create a replacement.
+
+### Configure firewall rules
+
+Next, you configure GCP firewall rules to allow inbound traffic from the
+on-premises network subnets. You must also configure the on-premises network
+firewall to allow inbound traffic from your VPC subnet prefixes.
+
+1. In the GCP Console,
+[go to the GCP Firewall rules page](https://console.cloud.google.com/networking/firewalls).
+1. Click **Create firewall rule**.
+1. Populate the following fields:
+
+1. **Name**—A name such as `vpnrule1`.
+1. **Network**—The name of the VPC network that you created
+    previously (for example,  `vpn-vendor-test-network`).
+1. **Target tags:**—All instances in the network  
+1. **Source filter**—A filter to apply your rule to specific sources of
+    traffic. In this case, choose source IP ranges.
+1. **Source IP ranges**—The on-premises IP ranges to accept from the
+    on-premises VPN gateway.
+1. **Allowed protocols and ports**—The string `tcp;udp;icmp`.
+
+1. Click **Create**.
+
+## Configure the Alibaba Cloud side
+
+This section includes sample tasks that describe how to configure the
+on-premises side of the VPN gateway configuration using Alibaba Cloud VPN Gateway.
+
+### Create an Alibaba Cloud VPC
+This section covers the steps of creating an Alibaba Cloud VPC.
+
+1. Login to the Alibaba Management Console
+    - Go to **Products > Virtual Private Cloud** 
+1. Create a new VPC
+    - Select a **Region**  (for example, US (Silicon Valley))
+    - Click the **Create VPC** button
+1. Configure the following VPC settings:
+    - **VPC Name**—Enter the name of the VPC (for example, `alibaba-vpc`)
+    - **Destination CIDR Block**—Specify the IP address range for the VPC in the form of a Classless Inter-Domain Routing (CIDR) block (for example, `192.168.0.0/16`)
+
+
+### Create an Alibaba Cloud VSwitch
+This section covers the steps of configuring the Alibaba Cloud VSwitch.
+
+1. Within the same VPC configuration window pane, configure the following VSwitch settings:
+    - **VSwitch Name**—Enter the name of the VSwitch (for example, `alibaba-vswitch`)
+    - **Zone**—Select a zone for the VSwitch (for example, `USA West 1 Zone A`)
+    - **Destination CIDR Block**—Specify a subnet from the VPC’s CIDR IP address range (for example, `192.168.1.0/24`)
+1. Click on **OK** followed by the **Complete** button
+1. Verify the VPC and VSwitch status indicate “Available”
+
+### Create Alibaba Cloud VPN Gateways
+This section covers the steps of configuring the Alibaba Cloud VPN Gateway.
```

Comment:
Combine these 2 sentences:
This section covers the steps of configuring a single Alibaba Cloud VPN Gateway.